defaults/main.yml

```diff
@@ -131,7 +132,6 @@ airflow_config_template: "templates/airflow.cfg.j2"
 airflow_defaults_config:
   core:
     airflow_home_mode: "0700"
-    dags_repos_folder: "{{ airflow_user_home_path ~ '/airflow/dags_repos' }}"
     dags_folder: "{{ airflow_user_home_path ~ '/airflow/dags' }}"
```

Comment:
Looks like `airflow_deafult_config.core.dags_folder is redundant with the playbook variable you are removing. Should we be using that?

Reply:
@dkinzer Those two paths are slightly different, so it is not redundant.  This allows us to keep the current functionality without having to rewrite tasks and ensures that files go to the directories that we intend them to go to.